WalletWasabi.Tests/UnitTests/Services/Tor/TorStatusCheckerTests.cs

```diff
@@ -0,0 +1,508 @@
+					  "informational": false,
+					  "resolvedAt": "<no value>",
+					  "affected": ["metrics.torproject.org"],
+					  "filename": "2025-04-10-bridgedb-graphs-on-metrics-website.md"
+					}
+
+					]
+			      },
+			      {
+			        "name": "ExoneraTor",
+			        "description": "Queryable database of IP addresses which were a Tor relay at a specified date.",
+			        "category": "Tor Project websites",
+
+			        "status": "ok",
+
+			        "unresolvedIssues": []
+
+			      },
+			      {
+			        "name": "weather.torproject.org",
+			        "description": "The weather.torproject.org website, providing a service for relay operators to keep track of their relays.",
+			        "category": "Tor Project websites",
+
+			        "status": "ok",
+
+			        "unresolvedIssues": []
+
+			      },
+			      {
+			        "name": "donate.torproject.org",
+			        "description": "The donate.torproject.org website, used to donate funds to the Tor Project.",
+			        "category": "Tor Project websites",
+
+			        "status": "ok",
+
+			        "unresolvedIssues": []
+
+			      },
+			      {
+			        "name": "deb.torproject.org",
+			        "description": "The Debian archive",
+			        "category": "Tor Project websites",
+
+			        "status": "ok",
+
+			        "unresolvedIssues": []
+
+			      },
+			      {
+			        "name": "GitLab",
+			        "description": "Our code-review, bug tracking, and collaboration platform of choice (gitlab.torproject.org).",
+			        "category": "Internal systems",
+
+			        "status": "ok",
+
+			        "unresolvedIssues": []
+
+			      },
+			      {
+			        "name": "Email",
+			        "description": "General email delivery from within the Tor Project.",
+			        "category": "Internal systems",
+
+			        "status": "ok",
+
+			        "unresolvedIssues": []
+
+			      },
+			      {
+			        "name": "Mailing lists",
+			        "description": "Our public and private mailing lists (lists.torproject.org) powered by Mailman, the GNU Mailing List Manager.",
+			        "category": "Internal systems",
+
+			        "status": "ok",
+
+			        "unresolvedIssues": []
+
+			      },
+			      {
+			        "name": "DNS",
+			        "description": "Domain name resolution for the Tor Project infrastructure.",
+			        "category": "Internal systems",
+
+			        "status": "ok",
+
+			        "unresolvedIssues": []
+
+			      }
+			  ],
+
+
+			  "buildDate": "2025-05-01",
+			  "buildTime": "15:22",
+			  "buildTimezone": "UTC",
+			  "colorBrand": "#7D4698",
+			  "colorOk": "#158474",
+			  "colorDisrupted": "#B35E0F",
+			  "colorDown": "#CA3D46",
+			  "colorNotice": "#157BB2",
+			  "alwaysKeepBrandColor": "true",
+			  "logo": "https://status.torproject.org/logo.png",
+			  "googleAnalytics": "UA-00000000-1"
+			}
+			""";
+
+		var deserialized = JsonDecoder.FromString(jsonResponseWithIssues, Decode.TorStatus);
+		Assert.NotNull(deserialized);
```

Comment:
Asserting that it is not null is not enough, assert that the deserialized instance contains what it should contains.